resources/base.scss

```diff
@@ -86,7 +86,7 @@ th {
 .form-area {
     display: inline-block;
     background: $color_primary5;
-    padding: 5px 10px 10px;
+    padding: 0px 10px 10px 0px;;
```

Comment:
Extra semicolon. Also, there are several items with class `form-area`, can you confirm they still look correct?